docs/blueprints/storagenode-graceful-exit/overview.md

```diff
@@ -35,7 +35,7 @@ The design is divided into four parts:
 
 Overall a good graceful exit process looks like:
 
-1. Storage node Operator initiaties the graceful exit process, which:
+1. Storage node Operator initiatives the graceful exit process, which:
```

Comment:
This should be "initiates".

Reply:
Ouch! I will revert it.

Reply:
Oh! You reverted it back to what it was. But you were correct that it was a misspelling: "initiaties" is wrong, you just had the wrong replacement word. Should be "initiates" instead of "initiatives".